ruby/red-arrow/Gemfile

```diff
@@ -23,6 +23,7 @@ gemspec
 
 gem "benchmark-driver"
 gem "bundler"
+gem "csv"
```

Comment:
Could you add this to https://github.com/apache/arrow/blob/main/ruby/red-arrow/red-arrow.gemspec instead of here?
Because `csv` is a run-time dependency not development dependency.

Reply:
Thanks！